common/middleware/instrument.go

```diff
@@ -39,14 +40,32 @@ func (i Instrument) Wrap(next http.Handler) http.Handler {
 	})
 }
 
+var invalidChars = regexp.MustCompile(`[^a-zA-Z0-9]+`)
```

Comment:
Hmm looking into this, there appears to be no restrictions on label values.  @peterbourgon do you remember why this existed to begin with?